sphinx_search/static/js/rtd_sphinx_search.js

```diff
@@ -124,6 +118,18 @@ const updateSearchBar = () => {
 };
 
 
+/*
+ * Returns true if the modal window is visible.
+ */
+const isModalVisible = () => {
+  let modal = document.querySelector(".search__outer__wrapper");
+  if (modal !== null && modal.style !== null && modal.style.display !== null) {
+    return style.display === 'block';
```

Comment:
`style` doesn't seem to be defined. if not, i assume this is the fix:

```suggestion
    return modal.style.display === 'block';
```